fixtures/webview.py

```diff
@@ -623,3 +623,25 @@ def aws_secret_access_key_value(request):
     awssecretvalue = config.getoption("aws_secret_access_key_value")
     if awssecretvalue is not None:
         return awssecretvalue
+
+
+@pytest.fixture
+def code_tag(request):
+    """Return a deployment code tag"""
+    config = request.config
+    code_tag = config.getoption("code_tag") or config.getini("code_tag")
+    if code_tag is not None:
+        skip_if_destructive_and_sensitive(request, code_tag)
```

Comment:
Will ` skip_if_destructive_and_sensitive` ever do anything in these two fixtures given it looks for specific URL structures (assuming I understand it correctly)?